pkgs/by-name/de/deezer-enhanced/package.nix

```
@@ -0,0 +1,116 @@
+{
+  ### Tools
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  makeWrapper,
+  autoPatchelfHook,
+  dpkg,
+  gnutar,
+
+  ### Libs
+  xorg,
+  libxkbcommon,
+  glib,
+  nss,
+  dbus,
+  at-spi2-atk,
+  cups,
+  gtk3,
+  pango,
+  cairo,
+  expat,
+  systemdLibs,
+  alsa-lib,
+  nwjs,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "deezer-enhanced";
+  version = "1.2.0";
+
+  src = fetchurl {
+    url = "https://github.com/duzda/${pname}/releases/download/v${version}/${pname}_${version}_amd64.deb";
+    hash = "sha256-oxdc+FOpomcEt7gXsUiN3MCweVaOWxidefGXT33YVcw=";
+  };
+
+  nativeBuildInputs = [
+    ### To unpack deezer-enhanced
+    dpkg
+    gnutar
+  ];
+
+  buildInputs = [
+    ### Tools
+    makeWrapper
+    autoPatchelfHook
```

```suggestion
  nativeBuildInputs = [
    ### To unpack deezer-enhanced
    dpkg
    gnutar

    makeWrapper
    autoPatchelfHook
  ];

  buildInputs = [
```
Since they're not used in the packages runtime, no need to have them in buildInputs